cmake/linux/lmms

```diff
@@ -1,4 +1,4 @@
-?package(lmms):needs="X11" section="Apps/Sound" \
+?package(lmms):needs="X11" section="Applications/Sound" \
   title="LMMS" hints="Audio" command="/usr/bin/lmms" \
   longtitle="LMMS" \
-  icon="/usr/share/pixmaps/lmms.png"
+  icon32x32="/usr/share/pixmaps/lmms.xpm"
```

Comment:
I'd recommend we hard code as little of this as possible and `CONFIGURE` it.
 * `lmms` in `/usr/bin/lmms` can be assumed
 * Longtitle can be assumed
 * Icon is a dupe of `/usr/share/pixmaps/lmms.xpm` defined in `cmake/linux/CMakeLists.txt`